students/14th/eommoonjoo/style/login.css

```
@@ -0,0 +1,103 @@
+* { box-sizing: border-box;}
```

```suggestion
* {
 box-sizing: border-box;
 }
```
- 속성은 이런식으로 작성해주시면 좋을 듯 합니다 :)